docs/concepts/master-key-generation.md

```diff
@@ -0,0 +1,170 @@
+---
+order: 2
+page:
+  headHtml: |
+    <snippet var="js.base" />
+    <snippet var="js.prism" />
+---
+
+# Master Key Generation
+
+_Master key generation_ is the process by which the wallet turns a [[recovery-phrases|seed]] (_entropy_) into 
+a secure cryptographic key. Child keys can be derived from a master key to produce
+a derivation tree structure as outlined in [[hierarchical-deterministic-wallets]].
+
+In Cardano, the master key generation algorithm is different depending on which style of wallet
+one is considering. In each case however, the generation is a function from an initial
+seed to an extended private key (_XPrv_) composed of:
+
+- 64 bytes: an extended Ed25519 secret key composed of:
+    - 32 bytes: Ed25519 curve scalar from which few bits have been tweaked (see below)
+    - 32 bytes: Ed25519 binary blob used as IV for signing
+- 32 bytes: chain code for allowing secure child key derivation
+
+::: {.info-block}
+### Additional resources
+
+- [SLIP 0010](https://github.com/satoshilabs/slips/blob/master/slip-0010.md)
+- [BIP 0032](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)
+- [BIP 0039](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki)
+- [RFC 8032](https://tools.ietf.org/html/rfc8032#section-5.1.5)
```

Comment:
```suggestion
- [SLIP 0010](https://github.com/satoshilabs/slips/blob/master/slip-0010.md)
- [BIP 0032](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)
- [BIP 0039](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki)
- [RFC 8032](https://tools.ietf.org/html/rfc8032#section-5.1.5)
- [CIP 3 — "Wallet key generation"](https://cips.cardano.org/cips/cip3/)
- [CIP 1852 — "HD (Hierarchy for Deterministic) Wallets for Cardano"](https://cips.cardano.org/cips/cip1852/)
```

While we're at it, it makes sense to add the relevant Cardano Improvement Proposals (CIPs) as well.